app/controllers/api/projects.php

```diff
@@ -260,11 +262,34 @@
                         'date' => \strtotime($point['time']),
                     ];
                 }
+
+                // Realtime Connections
+                $result = $database->query('SELECT sum(value) AS "value" FROM "appwrite_usage_realtime_clients" WHERE time > \''.$start.'\' AND time < \''.$end.'\' AND "metric_type"=\'counter\' AND "project"=\''.$project->getId().'\' GROUP BY time('.$period[$range]['group'].') FILL(null)');
```

Comment:
These queries will need to be part of the usage worker i believe . Are we planning to do this refactor after the release?

Reply:
Yes, once 0.10.0 is out - I will need to refactor lots of stuff to adapt to the DB refactor and new usage.